docs/docsite/rst/reference_appendices/release_and_maintenance.rst

```diff
@@ -408,7 +408,7 @@ Since Ansible is a package of individual collections, the deprecation cycle depe
 ansible-core deprecation cycle
 -------------------------------
 
-The deprecation cycle in ``ansible-core`` is normally across 4 feature releases (2.x. where the x marks a feature release). The feature is normally removed in the 4th release after we announce the deprecation. For example, something deprecated in 2.10 will be removed in 2.13. The tracking is tied to the number of releases, not the release numbering itself. Although this is the standard, there are times where a deprecation cycle for a feature or behavior may have a longer or shorter deprecation cycle based on use or urgency of removal.
+The deprecation cycle in ``ansible-core`` is normally across 4 feature releases (2.x. where the x marks a feature release). The feature is normally removed in the 4th release after we announce the deprecation. For example, something deprecated in 2.10 will be removed in 2.13. The tracking is tied to the number of releases, not the release numbering itself. Although this is the standard, there are times where a deprecation cycle for a feature or behavior may have a longer or shorter deprecation cycle based on use or urgency of removal. Unintended or undocumented functionality may be removed without a deprecation cycle.
```

Comment:
@oraNod yeah, it's complicated.  I do specifically mean `available through happenstance` in this case.  We don't cover the scenario of bug fixing changing behavior, because fixing bugs always changes behavior, even though someone may have been dependending on the broken behavior.

I don't think stating `unsupported` gives the right meaning.  The original recommendation made to me was just "undocumented" and not including "unintended" but I felt that someone could argue that a weird confluence of wording made it seem as though an unintended feature was documented, so I added that "unintended" part there.

I'm open to suggestions about how we could describe "unintended" better, whether that is just stating "Functionality available by happenstance" in maybe some other way 🙂

Reply:
Maybe we could just go with "Unintended functionality, which is available through happenstance, or undocumented functionality..."?

That's more verbose but removes some ambiguity imo. Perhaps `s/through happenstance/accidentally` would be better. @samccann WDYT?

Reply:
> I'm open to suggestions about how we could describe "unintended"

Accidental feature? Be sure to link https://www.hyrumslaw.com there :)

Reply:
Right, looking at my second comment again with (sort of) fresh eyes and it's not great. I do think there's a thin line between an unintended feature and a bug. My thoughts were to increase the distinction between those two things. "by happenstance" doesn't quite fit because that usually implies good fortune. "accidentally", well...

I've done a bit of reading but I'm not sure there are many clear options to express what I had in mind. Maybe we could just explicitly say this doesn't include/refer to bugs or flaws. But maybe it's better not to overcomplicate the wording.

Hopefully there is some value in these comments somewhere and I'm not making it too difficult. Cheers.

Reply:
@sivel Hey man, we talked about this at the docs / DaWGs meeting today. Actually it was kind of cool because someone from the community had a really good example of unintended functionality. Anyway we didn't come to a definitive answer on wording. I feel like this is the best way to express what I had in mind:

```
Unintended or undocumented functionality may be removed without a deprecation cycle. In this context, unintended functionality refers specifically to emergent features that occur outside the release roadmap rather than a bug or design flaw.
```

That's a lot of extra words to explain something that we might not even really need to clarify. As @samccann pointed out keeping it vague or slightly ambiguous could also be beneficial because it gives more flexibility. I'll leave it to you to decide anyway. Thanks!

Reply:
Whether the functionality is or was _intentional_ is irrelevant. So, suggest dropping the allusion to intent.

Likewise for _functionality_; it's just as likely to be a _dysfunction_, a quirky _side-effect_, or something nobody has ever noticed.

Further, such a change may not be a _removal_. It could be any of a long list of different types of changes.

I suggest, "Undocumented behavior may be changed without a deprecation cycle." It's both shorter and more accurate.

Also, this was discussed - with relevant example - in this morning's [DaWGs meeting](https://forum.ansible.com/t/meeting-log-ansible-docs-2024-03-19-1628/4459). ([full_log.txt](https://forum.ansible.com/uploads/short-url/3Hk6l95LY7Nk9xs4u5lFmLiZKB2.txt))